### PR TITLE
Update README with basic site management and launch instructions.

### PR DESCRIPTION
**Pull request by trackleft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,55 @@
...
+When you are ready to launch your site, you can remove dev dependencies included with Quickstart and this scaffolding repository with composer, but first you will want to make sure the development modules are not enabled in Drupal.
+
+Example
+```
+drush pm-uninstall -y devel migrate_devel config_inspector
+```
+
+Once uninstalled, dev dependencies can safely be removed with composer. 
+```
+composer install --no-dev
```
Comment (trackleft) [inline README.md:61]:
```suggestion
composer remove --dev
```

File: README.md [hunk 2]
```diff
@@ -8,11 +8,54 @@
 - Utilizes [composer-installers](https://github.com/composer/installers) and [Drupal Composer Scaffold](https://github.com/drupal/core-composer-scaffold) plugins
 - Can be used as project template for composer [create-project command](https://getcomposer.org/doc/03-cli.md#create-project):
   ```
-  composer create-project az-digital/az-quickstart-scaffolding:2.0.x-dev azqs --no-interaction --no-dev
+  composer create-project az-digital/az-quickstart-scaffolding:2.2.x-dev azqs --no-interaction --no-dev
```
Comment (trackleft) [inline README.md:11]:
```suggestion
  composer create-project az-digital/az-quickstart-scaffolding:2.3.x-dev azqs --no-interaction --no-dev
```

File: README.md [hunk 3]
```diff
@@ -8,11 +8,54 @@
 - Utilizes [composer-installers](https://github.com/composer/installers) and [Drupal Composer Scaffold](https://github.com/drupal/core-composer-scaffold) plugins
 - Can be used as project template for composer [create-project command](https://getcomposer.org/doc/03-cli.md#create-project):
   ```
-  composer create-project az-digital/az-quickstart-scaffolding:2.0.x-dev azqs --no-interaction --no-dev
+  composer create-project az-digital/az-quickstart-scaffolding:2.3.x-dev azqs --no-interaction --no-dev
```
[the remark this answers is not in the document]
Comment (trackleft) [reply README.md:11]: Should this one be `dev-main`?

[the remark this answers is not in the document]
Comment (joeparsons) [reply README.md:11]: I guess we probably don't want to recommend to anyone that they start their project with `dev-main`/`2.4.x` so maybe it make more sense to reference the latest "stable" release branch in main (like you already are).

**Review by joeparsons — DISMISSED** [review]

**Review by cpoff — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -8,11 +8,54 @@
...
+Arizona Digital has added a [README for migrating into Quickstart 2](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_migration/README.md).
+
+# Updating Quickstart
+
+Currently, Arizona Digital supports the two most recent [minor releases of Arizona Quickstart](https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md).
+
+- `composer install` will install updates and pull in dev dependencies, and also apply patches if they exist.
+- `composer install  --no-dev` will install updates, and will remove dev dependencies, and also apply patches if they exist.
+- `composer update` is supposed to get the latest available based on the [version constraints](https://getcomposer.org/doc/articles/versions.md#summary) in your composer.json file, and also apply patches if they exist.
+- `composer require "az-digital/az_quickstart:2.4.0" --update-no-dev` Will pin to a specific version of quickstart without dev dependencies. You can update your site, incrementing the version number whenever you want to update, or use version constraints with `composer update --no-dev` to only update to a tagged version.
```
Comment (bberndt-uaz) [inline README.md:26]:
```suggestion
- `composer require "az-digital/az_quickstart:2.4.0" --update-no-dev` Will pin to a specific version of Quickstart without dev dependencies. You can update your site, incrementing the version number whenever you want to update, or use version constraints with `composer update --no-dev` to only update to a tagged version.
```

**Review by bberndt-uaz — APPROVED** [review]

**Review by bberndt-uaz — APPROVED** [review]

**Review by cpoff — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Pull request merged**